### PR TITLE
Add Chase to pubs

### PR DESCRIPTION
**Pull request by jaywonchung**
This workshop is not under ICLR, but has its own org behind it (This year it's with ICLR, but last year it was with NeurIPS). That's why I made the venue key just `ClimateML` instead of something like `ClimateML-ICLR`.

#177 is to be kept open for the camera ready version, slides, and code.

### CODE REVIEW

File: source/publications/index.md [hunk 1]
```diff
@@ -286,6 +286,13 @@ venues:
       date: 2022-07-17
       url: https://icml.cc/Conferences/2022
       acceptance: 21.94%
+  ClimateML:
+    category: Workshops
+    occurrences:
+    - key: ClimateML-ICLR'23
```
Comment (insujang) [inline source/publications/index.md:292]:
```suggestion
  ClimateChangeAI:
    category: Workshops
    occurrences:
    - key: CCAI-ICLR'23
```

File: source/_data/SymbioticLab.bib [hunk 2]
```diff
@@ -1338,3 +1338,17 @@ @Article{flint:arxiv23
   }
 }
 
+@InProceedings{chase:climateml-iclr23,
+  author    = {Yang, Zhenning and Meng, Luoxi and Chung, Jae-Won and Chowdhury, Mosharaf},
+  booktitle = {Tackling Climate Change with Machine Learning},
+  title     = {Chasing Low-Carbon Electricity for Practical and Sustainable {DNN} Training},
+  year      = {2023},
+
+  publist_confkey = {ClimateML-ICLR'23},
```
Comment (insujang) [inline source/_data/SymbioticLab.bib:1347]:
```suggestion
  publist_confkey = {CCAI-ICLR'23},
```

**Review by insujang — CHANGES_REQUESTED** [review]
The official organization name and workshop name is Climate Change AI (CCAI). Maybe better to use it?

**Comment by jaywonchung** [comment]
That's much better. Thanks for the suggestion! I've applied both.

**Review by insujang — APPROVED** [review]
LGTM

**Comment by mosharaf** [comment]
If #177 is open, why did we merge this instead of the arxiv one?

**Comment by jaywonchung** [comment]
This is the arXiv version. I added this quick and merged this PR to publish Chase to the list of publications as early as possible. I left #177 open to serve as a reminder to update the paper link to the official one when the workshop goes live.

**Comment by mosharaf** [comment]
This should've been the arxiv link and not like this, like all other papers that have had arxiv versions. This is breaking the protocol that we've implicitly created re:arxiv and conference versions

**Comment by mosharaf** [comment]
We have never uploaded a conference/workshop paper that's not camera-ready. Whenever we did early upload, they've always been arxiv separate from a later camera-ready version

**Comment by jaywonchung** [comment]
I think I get what's wrong. Let me post a PR that amends this now.

**Comment by insujang** [comment]
Does that mean Zeus should also have arXiv link for now? I approved this since I saw Zeus also has its own PDF uploaded so that arXiv link should only be applied when the paper has explicitly been tagged as an arXiv paper, not ones accepted but not alive.

**Comment by jaywonchung** [comment]
This one's my bad. The PDF up on Zeus is actually the camera ready version we submitted to NSDI, so it'll never change.

**Comment by insujang** [comment]
Oh okay. My applogies for the confusion.

**Comment by mosharaf** [comment]
Thanks both of you. The arxiv version and camera-ready version of a paper may or may not be the same. So it's better to keep them separate

**Comment by mosharaf** [comment]
For example, Oort is a good example. Arxiv one is a super set of OSDI version.

**Pull request merged**